WSL/toc.yml

```diff
@@ -0,0 +1,50 @@
+- name: Overview
+  href: overview.md
+  items:
+    - name: About
+      href: about.md
+    - name: WSL1 and WSL2
+      href: wsl1-wsl2.md
+- name: Getting Started
+  href:
+  items:
+    - name: Install WSL1
```

Comment:
Suggestion: Swap this page to be the 2nd item and rename it to: `Install just WSL1`